docs/development/ipv6.md

```diff
@@ -7,8 +7,6 @@
 [GEP-21](../proposals/21-ipv6-singlestack-local.md) proposes IPv6 Single-Stack Support in the local Gardener environment.
 This documentation will be enhanced while implementing GEP-21, see [gardener/gardener#7051](https://github.com/gardener/gardener/issues/7051).
 
-To use IPv6 single-stack networking, the [feature gate](../deployment/feature_gates.md) `IPv6SingleStack` must be enabled on gardener-apiserver and gardenlet.
```

Comment:
> Infrastructure-specific validations will be added in parallel to the corresponding provider extensions.

Should we add this information to this document?

Also, should we add a "support matrix" that shows whether networking extensions support IPv6 and dual-stack networking and start from which version?

Reply:
I understand why it might be interesting to have this feature matrix, but I am hesitant to add it here. The functionality will be added to other repositories. Hence, it will likely be forgotten to update this here.

Not sure where a good place would be, though. Ideas are welcome.

Reply:
True. We could state that you need to check if your provider and networking extension support IPv6/dual-stack. That's probably the next best thing.

Reply:
Adapted as requested in https://github.com/gardener/gardener/pull/10716/commits/7a13c0d74b842edeff56cf29da6cda529a76bba0.

WDYT?